demos/functions/views/demo/workbox-streams/sw.hbs

```diff
@@ -0,0 +1,44 @@
+{{{WORKBOX_SW_IMPORT}}}
+
+const CACHE_NAME = 'my-cache';
+const START_CACHE_KEY = 'start';
+const END_CACHE_KEY = 'end';
+
+self.addEventListener('install', (event) => {
+  event.waitUntil((async () => {
+    const cache = await caches.open(CACHE_NAME);
+    await Promise.all([
+      cache.put(START_CACHE_KEY, new Response('<html><head></head><body>')),
```

Comment:
Would be preferable to make this a little more "fun".

Perhaps you could have an iframe that displays a HTML page indicating its not streamed and with SW installed it loads a HTML page with a header and footer with a party popper in the middle. This highlights a clear use case of header and footer and party popper highlights its working.

Reply:
Updated.